src/corona_hakab_model/update_matrix.py

```diff
@@ -54,9 +74,25 @@ def change_connections_policy(self, connection_types_to_use: Iterable[Connection
         self.matrix.set_factors(factors)
         self.normalize()
 
-    def update_matrix_step(self):
+    def update_matrix_step(self, circle_policies : Iterable[PolicyByCircles]):
         """
         Update the matrix step
         """
         # for now, we will not update the matrix at all
-        pass
+        for policy_by_circle in circle_policies:
```

Comment:
Like I wrote in slack,
I'd move this implementation to a seperate function in this class called "apply_circle_policies" or so,
As there will be more stages in the updating of the matrix, and this function will eventually call each of them one by one...